nixos/modules/services/x11/desktop-managers/plasma5.nix

```diff
@@ -182,16 +182,6 @@ in
 
       environment.variables = {
         # Enable GTK applications to load SVG icons
```

Comment:
Is this comment still relevant (asking because I don't know) now that L185 - L194 is moved to the derivation?